### PR TITLE
Update DeletionRecovery Level

### PR DESCRIPTION
**Pull request by jlichwa**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Contribution checklist:
- [ ] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [ ] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [ ] Service team MUST add the "**WaitForARMFeedback**" label if the management plane API changes fall into one of the below categories. 
  - adding/removing APIs.
  - adding/removing properties.
  - adding/removing API-version. 
  - adding a new service in Azure.

<i>Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.</i>

- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Comment by jlichwa** [comment]
Please #hold-off it is still a draft

File: specification/keyvault/data-plane/Microsoft.KeyVault/stable/7.0/keyvault.json [hunk 1]
```diff
@@ -3879,11 +3879,44 @@
...
+            "CustomizedRecoverable+ProtectedSubscription"
           ],
           "x-ms-enum": {
             "name": "DeletionRecoveryLevel",
-            "modelAsString": true
+            "modelAsString": true,
+            "values": [
+              {
+                "value": "Purgeable",
+                "description": "It can be purged by a priviliged user after 90 days"
```
Comment (heaths) [inline specification/keyvault/data-plane/Microsoft.KeyVault/stable/7.0/keyvault.json:3893]: Nit: these are sentences used in documentation and should use punctuation, i.e. end in a period.

File: specification/keyvault/data-plane/Microsoft.KeyVault/stable/7.0/keyvault.json [hunk 2]
```diff
@@ -3879,11 +3879,44 @@
...
+              },
+              {
+                "value": "Recoverable+Purgeable",
+                "description": "It can be recovered by a priviliged user until is purged.It can be purged by a priviliged user after 90 days"
```
Comment (heaths) [inline specification/keyvault/data-plane/Microsoft.KeyVault/stable/7.0/keyvault.json:3897]: Space between "purged.It".

File: specification/keyvault/data-plane/Microsoft.KeyVault/stable/7.0/keyvault.json [hunk 3]
```diff
@@ -3879,11 +3879,44 @@
...
+              },
+              {
+                "value": "Recoverable",
+                "description": "It can be recovered by priviliged user until is purged. System purge after 90 days"
+              },
+              {
+                "value": "Recoverable+ProtectedSubscription",
+                "description": ""
```
Comment (heaths) [inline specification/keyvault/data-plane/Microsoft.KeyVault/stable/7.0/keyvault.json:3905]: Missing description.

File: specification/keyvault/data-plane/Microsoft.KeyVault/stable/7.0/keyvault.json [hunk 4]
```diff
@@ -3879,11 +3879,44 @@
...
+              },
+              {
+                "value": "CustomizedRecoverable+Purgeable",
+                "description": "It can be recovered by a priviliged user until is purged. It can be purged by a priviliged user after number of days specified in 'softDeleteRetentionInDays' "
+              },
+              {
+                "value": "CustomizedRecoverable",
+                "description": "It can be recovered by priviliged user until is purged. System purge after number of days specificed in 'softDeleteRetentionInDays'"
```
Comment (heaths) [inline specification/keyvault/data-plane/Microsoft.KeyVault/stable/7.0/keyvault.json:3913]: "*a* privileged user"

File: specification/keyvault/data-plane/Microsoft.KeyVault/stable/7.0/keyvault.json (hunk 1 above)
Comment (heaths) [inline specification/keyvault/data-plane/Microsoft.KeyVault/stable/7.0/keyvault.json:3893]: "privileged" (last "i" -> "e"). If you're using Visual Studio Code, I highly recommend the cSpell and Micorosft Docs extensions (the latter is an extension pack).

**Review by heaths — APPROVED** [review]

**Comment by jlichwa** [comment]
#sign-off

**Comment by jlichwa** [comment]
#sign-off

**Review by jhendrixMSFT — APPROVED** [review]

**Pull request merged**